src/mmrelay/meshtastic/events.py

```
@@ -0,0 +1,945 @@
+    STALE_DISCONNECT_TIMEOUT_SECS,
+    STARTUP_PACKET_DRAIN_SECS,
+)
+
+__all__ = [
+    "on_lost_meshtastic_connection",
+    "on_meshtastic_message",
+    "reconnect",
+]
+
+
+def on_lost_meshtastic_connection(
```

Ruff (0.15.9)

[warning] Too many branches (31 > 12)

(PLR0912)